store-implementation/map-store/src/test/java/uk/gov/gchq/gaffer/mapstore/impl/GetAdjacentIdsTest.java

```diff
@@ -280,4 +281,39 @@ public void testGetElementsByEntityIdWithViewRestrictedByGroupAndAPostAggregatio
         expectedResults.remove(new EntitySeed("Y2"));
         assertEquals(expectedResults, resultsSet);
     }
+
+    @Test
+    public void shouldFailValidationWhenEntityHasFilter() throws OperationException {
```

Comment:
Please could you also write some unit tests to specifically check the validation doesn't fail when there are entities, but without filters.